source/userguide/kb.rst

```diff
@@ -701,3 +701,88 @@ In the example below, we search for restaurants whose names best match ``firetra
 .. note::
 
    We can set the ``size`` parameter of the :meth:`execute()` method to specify the maximum number of records.
+
+
+Dealing with unstructured text
+------------------------------
+The knowledge bases described in the previous sections are comprised of clearly defined data types whose pattern makes them easily searchable. We call them structured data.
```

Comment:
```suggestion
The knowledge bases described in the previous sections are comprised of clearly defined data types whose format makes them easily searchable. We call them structured data.
```

Reply:
Wanted to find a better word than "pattern".

Alternately, "The knowledge bases described in the previous sections are comprised of clearly defined data types that are easily searchable. We call them structured data."